tests_pytest/_test_util/tpc_util.py

```diff
@@ -50,12 +50,11 @@ def build_mp_config_options_for_kernel_bias_ops(base_w_config: AttributeQuantiza
     return QuantizationConfigOptions(quantization_configurations=mp_configs, base_config=base_op_config)
 
 
-def minimal_tpc():
+def minimal_cfg_options():
```

Comment:
please publish this new util in the team's chat with a link to the relevant file in the PR